QuickStart/Info.plist

```diff
@@ -16,23 +16,16 @@
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
-	<key>CFBundleSignature</key>
-	<string>????</string>
-    <key>NSAppTransportSecurity</key>
-    <dict>
-        <key>NSAllowsArbitraryLoads</key>
-        <true/>
-    </dict>
 	<key>CFBundleURLTypes</key>
 	<array>
 		<dict>
 			<key>CFBundleTypeRole</key>
 			<string>Editor</string>
 			<key>CFBundleURLName</key>
-			<string>msquickstart</string>
+			<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>msquickstart</string>
+				<string>msal2a814505-ab4a-41f7-bd09-3fc614ac077c</string>
```

Comment:
Will remove. Yes previously we wanted developer to always be ready to support broker. I think we can let that go now since SSO isn't a scenario of the Quickstart. I do discuss the URIs to set in the [SSO doc for iOS](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-v1-enable-sso-ios).